lib/generators/react_on_rails/base_generator.rb

```diff
@@ -28,26 +28,23 @@ def create_react_directories
       end
 
       def copy_base_files
-        base_path = "base/base/"
         base_files = %w[app/controllers/hello_world_controller.rb
                         app/views/layouts/hello_world.html.erb
                         config/initializers/react_on_rails.rb
                         Procfile.dev
                         Procfile.dev-static]
-        base_files.each { |file| copy_file("#{base_path}#{file}", file) }
+        base_files.each { |file| copy_file("#{base_path}/#{file}", file) }
       end
 
       def copy_js_bundle_files
-        base_path = "base/base/"
         base_files = %w[app/javascript/packs/server-bundle.js
                         app/javascript/bundles/HelloWorld/components/HelloWorldServer.js
                         app/javascript/bundles/HelloWorld/components/HelloWorld.module.css]
-        base_files.each { |file| copy_file("#{base_path}#{file}", file) }
+        base_files.each { |file| copy_file("#{base_path}/#{file}", file) }
```

Comment:
why is this changing?

Reply:
To follow the same structure in creating the file path.

Reply:
I changed these changes back. I think it was better to extract the base_path and make all these path creation the same way, but that is not related to this PR so they are reverted back to what they were.

So this issue is resolved.